src/CabanaPD_Solver.hpp

```diff
@@ -191,12 +184,11 @@ class SolverElastic
                  ", Total neighbors: ", total_neighbors, "\n" );
             out.close();
         }
-        init_time += init_timer.seconds();
+        _init_timer.stop();
```

Comment:
@streeve What does  "_init_timer" measure here besides " _neighbor_timer"?

Reply:
In this case, almost nothing. But in the fracture case below it does more (precracks and broken bond allocation)